nf1_cellprofiler_classifier_fs_all_cellprofiler_all_feat_perfect.ipynb

```diff
@@ -0,0 +1,499 @@
+{
```

Comment:
<p>I recommend adding a title for this section (first six cells) like you do in the rest of the notebook to keep consistent.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/NF1_SchwannCell_data_analysis/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>